src/components/tree/Tree.tsx

```diff
@@ -62,30 +52,21 @@ const Tree: FC<Props> = ({ treeSections, treePath, stars }: Props) => {
         RootInfo | TrunkInfo | LeafInfo | BranchInfo
       ][]
     );
-    if (info.branch && treeSections.branch?.length > 0) {
-      setBranchesEntries(
-        Object.entries(info.branch.branches) as [
-          string,
-          { id: number; title: string }
-        ][]
-      );
-    }
   }, []);
 
   const keywords: Keywords = useMemo(() => {
     const keywords: Keywords = {
       root: [],
       trunk: [],
       leaf: [],
-      branch: {
-        branch_type_1: [],
-        branch_type_2: [],
-        branch_type_3: [],
-      },
+      branch_1: [],
+      branch_2: [],
+      branch_3: [],
     };
     const sections: Section[] = Object.keys(keywords) as Section[];
     for (let section of sections) {
-      if (section !== "branch") {
+      if (treeSections[section]?.length) {
```

Comment:
Oh just saw you had to ignore ts, I didn't pull the code to review locally before suggesting changing to `treeSections[section]?.length`, it's always good to do logic, instead of skipping ts check (`@ts-ignore`). I apologize that, I think it makes sense in the way you had it before.

BTW, there are many ways to do this where TS checks and execution will work just fine.

You could avoid the check of the length, by removing the condition `if (treeSections[section]?.length) {` and then do `for (const article of treeSections[section] ?? [])`, let me know if that works fine locally, and I also thought that `let` was required when using `for of`, but you could use `const` since the variable won't be mutated inside the `for`.

BTW, this is a just a comment about code styling mostly, not something that will be blocking the PR, if you want to apply the changes or not will be just fine.

Reply:
Hi @dani-vh, thank you very much for you comment. I think your solution is perfect and it has worked perfectly for me locally.